joint_states: "/placeholder"

publish_rate: 50.0

#joint_names: ["x_joint", "y_joint", "z_joint", "roll_joint", "pitch_joint", "yaw_joint"]
joint_names: ["x_joint", "y_joint", "yaw_joint", "wheel0_joint", "wheel1_joint", "wheel2_joint", "cameraholder_joint"]

p_gain_roll:  0.4
i_gain_roll:  4
d_gain_roll:  0.00
i_max_roll:  1
i_min_roll:  -1
anti_windup_roll: true

p_gain_pitch:  0.4
i_gain_pitch:  4
d_gain_pitch:  0.00
i_max_pitch:  1
i_min_pitch:  -1
anti_windup_pitch: true

p_gain_yaw: 0.4 #1.2
i_gain_yaw:  4
d_gain_yaw:  0.00
i_max_yaw:  1
i_min_yaw:  -1
anti_windup_yaw: true

p_gain_cameraholder: 0.4 #1.2
i_gain_cameraholder:  4
d_gain_cameraholder:  0.00
i_max_cameraholder:  1
i_min_cameraholder:  -1
anti_windup_cameraholder: true

p_gain_x: 0.6 #1.2
i_gain_x: 0.004
d_gain_x: 0.00
i_max_x: 1
i_min_x: -1
anti_windup_x: true

p_gain_y:  0.6 #1.2
i_gain_y:  0.004
d_gain_y:  0.00
i_max_y:  1
i_min_y:  -1
anti_windup_y: true

p_gain_z:  0.6
i_gain_z:  0.004
d_gain_z:  0.00
i_max_z:  1
i_min_z:  -1
anti_windup_z: true

position_limit_roll: 0.35
position_limit_pitch: 0.35
position_limit_yaw: 6.3
position_limit_cameraholder: 6.3

velocity_limit_roll: 0.7
velocity_limit_pitch: 0.7
velocity_limit_yaw: 1
velocity_limit_cameraholder: 1
velocity_limit_x: 0.5
velocity_limit_y: 0.5
velocity_limit_z: 0.5